app/models/event.rb

```diff
@@ -1,3 +1,6 @@
 class Event < ApplicationRecord
   belongs_to :user
+  has_many :event_participants
```

Comment:
You can remove the belongs to in this model. The belongs to is handled by the EventParticipant model.